ashlar/scripts/ashlar.py

```diff
@@ -64,8 +65,16 @@ def main_inner(argv):
         '-q', '--quiet', dest='quiet', default=False, action='store_true',
         help='suppress progress display'
     )
+    parser.add_argument(
+        '--version', dest='version', default=False, action='store_true',
+        help='print version'
+    )
     args = parser.parse_args(argv[1:])
 
+    if args.version:
+        print('ashlar {}'.format(pkg_resources.require('ashlar')[0].version))
```

Comment:
Why not just `import ashlar` at the top (you'll also need `from __future__ import absolute_import`. Or just `from .. import __version__ as pkg_version` or something like that.